src/main/java/hudson/maven/Maven3ProcessFactory.java

```diff
@@ -149,4 +149,8 @@ public boolean accept(File dir, String name) {
         }
     };
 
+    protected final String getLoggingPath(MavenInstallation mvn){
+        return (getLauncher().isUnix()?":":";")+mvn.getHome()+"/conf/logging";
+    }
+
```

Comment:
This is the key change in the pull request.  Instead of using `mvn.getHomeDir().getPath()`, it uses `mvn.getHome()`